docs/EN/Configuration/transition-qa.md

```diff
@@ -145,7 +144,7 @@ Here's the recommendation:
 
 Are you using a commercial closed-loop system like Insulet's Omnipod 5, Tandem's Control IQ, Medtronic's 780G, etc.? If you're happy, stay if you're not getting the desired results; read through the chapter on "Coming from iOS Loop" to see the main issues that Trio solves.
 
-## I Want to Try Trio. How Do I Proceed?
```

Comment:
I think this should remain it's own subheading rather than being moved into `## Coming From Commercial Closed-Loop Systems`, so I would recommend reverting this line.

Reply:
It looks odd in the side menu when it's a ## Heading as well. I changed the formatting so it avoids both issues of falling under Commercial Closed loop systems and looking out of place in the side menu. #6b4db1e77c900493debf4425bc7818cf6b7653da